velox/connectors/hive/HiveConnectorSplit.cpp

```diff
@@ -88,6 +89,16 @@ folly::dynamic HiveConnectorSplit::serialize() const {
     obj["properties"] = nullptr;
   }
 
+  if (rowIdProperties.has_value()) {
+    folly::dynamic rowIdObj = folly::dynamic::object;
+    rowIdObj["metadataVersion"] = rowIdProperties->metadataVersion;
+    rowIdObj["partitionId"] = rowIdProperties->partitionId;
+    rowIdObj["tableGuid"] = rowIdProperties->tableGuid;
+    obj["rowIdProperties"] = rowIdObj;
+  } else {
+    obj["rowIdProperties"] = nullptr;
```

Comment:
Do you need to specify obj["rowIdProperties"] is null? thanks!

Reply:
fixed by using getOrDefault